osu.Game/Overlays/Comments/CommentMarkdownContainer.cs

```diff
@@ -1,9 +1,17 @@
 ﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+#nullable disable
```

Comment:
Why? Please remove and apply `!` liberally as needed, or at least only apply locally to the nested class.